Sage/carbon
# feat(navigation bar): describe navigation bar using design tokens, update tests - FE - 4873

### Proposed behaviour

Describes navigation bar component using the tokens of the design.
Updates tests when the above changes are made.
Removes the bottom border for a dark variant.

### Current behaviour

Navigation bar component use the theme styled-component property.
Currently dark variant has border-bottom.

### Checklist




#### QA


### Additional context

![image](https://user-images.githubusercontent.com/44143630/151766842-404484a1-907c-40e0-9122-f3a548413aaf.png)

### Testing instructions

The correct appearance can be checked in the component: Navigation bar.
For theme: mint and aegean there should be no differences.
Use devtools to check if navigation bar use css variables.
